lib/properties-order/__tests__/fixtures/properties-have-same-name-4.css

```
@@ -0,0 +1,27 @@
+div {
+	position: fixed;
+	top: 0;
+	Top: $header-height;
+	A: 0;
+	b: 1;
+	C: 2;
+	d: 3;
+	E: 4;
+	f: 5;
+	G: 6;
+	h: 7;
+}
+
+div {
+	position: fixed;
+	Top: $header-height;
+	top: 0;
+	a: 0;
+	B: 1;
+	c: 2;
+	D: 3;
+	e: 4;
+	F: 5;
+	g: 6;
+	H: 7;
+}
```

```suggestion
div {
	position: fixed;
	top: 0;
	Top: 0;
}

div {
	position: fixed;
	Top: 0;
	top: 0;
}
```

1. Removed Sass.
2. Removed non-existing properties.